SI/Sem.04/Pract.04/Task02.cpp

```diff
@@ -0,0 +1,21 @@
+#include <iostream>
+
+using namespace std;
+
+bool IsEven(int num)
+{
+	if (num % 2 == 0)
```

Comment:
num % 2 == 0 is a boolean expression, so it returns a boolean value that you can directly return